### PR TITLE
Feature/2 rebranding

### PR DESCRIPTION
**Pull request by KirillovDenis**
Related to #2

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: neofs-http-gw
-Upstream-Contact: tech@nspcc.ru
+Upstream-Name: frostfs-http-gw
+Upstream-Contact: tech@frostfs.info
 Source: https://github.com/nspcc-dev/neofs-http-gw
```
Comment (alexvanin) [inline debian/copyright]:
```suggestion
Source: https://github.com/TrueCloudLab/frostfs-http-gw
```

File: debian/copyright [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 Files: *
-Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-http-gw project 
-           (https://github.com/nspcc-dev/neofs-http-gw/blob/master/CREDITS.md)
+Copyright: 2018-2022 True Cloud Lab (@TrueCloudLab), contributors of frostfs-http-gw project
+           (https://github.com/nspcc-dev/frostfs-http-gw/blob/master/CREDITS.md)
```
Comment (alexvanin) [inline debian/copyright]:
```suggestion
           (https://github.com/TrueCloudLab/frostfs-http-gw/blob/master/CREDITS.md)
```

File: README.md [hunk 3]
```diff
@@ -38,32 +38,32 @@ version      Show current version
...
 gRPC interface and you need to provide some node that it will connect to. This
 can be done either via `-p` parameter or via `HTTP_GW_PEERS_<N>_ADDRESS` and
 `HTTP_GW_PEERS_<N>_WEIGHT` environment variables (the gate supports multiple
-NeoFS nodes with weighted load balancing).
+FrostFS nodes with weighted load balancing).
 
-If you launch HTTP gateway in bundle with [neofs-dev-env](https://github.com/nspcc-dev/neofs-dev-env), 
+If you launch HTTP gateway in bundle with [frostfs-dev-env](https://github.com/TrueCloudLab/frostfs-dev-env), 
 you can get the IP address of the node in the output of `make hosts` command 
 (with s0*.neofs.devenv name).
```
Comment (alexvanin) [inline README.md:55]: I assume we are going to rename it to `s0*.frostfs.devenv`. Mention that explicitly in #2 or fix it here. There are plenty of dev-env links and usage of `$NEOFS_NODE`.

**Review by alexvanin — COMMENTED** [review]
- Update paths in `frostfs-http-gw.dirs`, `frostfs-http-gw.postinst`

Overall LGTM. In next PR let's refactor internal code.

File: README.md (hunk 3 above)
Comment (KirillovDenis) [reply README.md:55]: I thought we should wait for renaming in dev-env [here](https://github.com/TrueCloudLab/frostfs-dev-env/blob/1099cdcef885efbbec56bc869dc04a64696ada23/.env#L2)

File: debian/copyright [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-Upstream-Contact: tech@nspcc.ru
-Source: https://github.com/nspcc-dev/neofs-http-gw
+Upstream-Name: frostfs-http-gw
+Upstream-Contact: tech@frostfs.info
+Source: https://github.com/TrueCloudLab/frostfs-http-gw
 
 Files: *
-Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-http-gw project 
-           (https://github.com/nspcc-dev/neofs-http-gw/blob/master/CREDITS.md)
+Copyright: 2018-2022 True Cloud Lab (@TrueCloudLab), contributors of frostfs-http-gw project
```
Comment (alexvanin) [inline debian/copyright]: I think it is just 2022. /cc @realloc

File: debian/frostfs-http-gw.postinst [hunk 5]
```diff
@@ -21,16 +21,16 @@ set -e
...
         USERNAME=http
-        id -u neofs-$USERNAME >/dev/null 2>&1 || useradd -s /usr/sbin/nologin -d /srv/neofs_cache --system -M -U -c "NeoFS HTTP gateway" neofs-$USERNAME
+        id -u frostfs-$USERNAME >/dev/null 2>&1 || useradd -s /usr/sbin/nologin -d /srv/neofs_cache --system -M -U -c "FrostFS HTTP gateway" frostfs-$USERNAME
         if ! dpkg-statoverride --list /etc/neofs/$USERNAME >/dev/null; then
-            chown -f root:neofs-$USERNAME /etc/neofs/$USERNAME
-            chown -f root:neofs-$USERNAME /etc/neofs/$USERNAME/config.yaml || true
+            chown -f root:frostfs-$USERNAME /etc/neofs/$USERNAME
+            chown -f root:frostfs-$USERNAME /etc/neofs/$USERNAME/config.yaml || true
             chmod -f 0750 /etc/neofs/$USERNAME
             chmod -f 0640 /etc/neofs/$USERNAME/config.yaml || true
```
Comment (alexvanin) [inline debian/frostfs-http-gw.postinst]:
```suggestion
            chmod -f 0640 /etc/frostfs/$USERNAME/config.yaml || true
```

File: debian/frostfs-http-gw.postrm [hunk 6]
```diff
@@ -21,7 +21,7 @@ set -e
 
 case "$1" in
     purge)
-        rm -rf /srv/neofs_cache
+        rm -rf /srv/frostfs_cache
```
Comment (alexvanin) [inline debian/frostfs-http-gw.postrm:24]: `debian/frostfs-http-gw.dirs` still uses `srv/neofs_cache`.

File: debian/copyright (hunk 4 above)
Comment (realloc) [reply debian/copyright]: FrostFS development by TrueCloudLab started at 2022, so we need to add a line, not t  replace NSPCC.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**